src/components/PageSection/ElementsCarousel/CarouselItem/CarouselItem.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+type CarouselItemProps = {
+  id: number;
+  title: string;
+  poster: string | undefined;
+  rating: number;
+  year: number;
+};
+
+const StyledCard = styled.div`
+  position: relative;
+  min-width: 140px;
+  margin-right: 20px;
+`;
+
+const StyledCardImg = styled.img`
+  border-radius: 6px;
+  -webkit-box-shadow: 0px 8px 13px -5px ${(props) => props.theme.main.boxShadowColor};
+  -moz-box-shadow: 0px 8px 13px -5px ${(props) => props.theme.main.boxShadowColor};
+  box-shadow: 0px 8px 13px -5px ${(props) => props.theme.main.boxShadowColor};
+`;
```

Comment:
You can also simplify this solution:
```suggestion
const StyledCardImg = styled.img`
${({
    theme: {main}
  }) => `
   border-radius: 6px;
  -webkit-box-shadow: 0px 8px 13px -5px ${main.boxShadowColor};
  -moz-box-shadow: 0px 8px 13px -5px ${main.boxShadowColor};
  box-shadow: 0px 8px 13px -5px ${main.boxShadowColor};
  `}
`;
```